HeterogeneousCore/AlpakaInterface/interface/Backend.h

```diff
@@ -0,0 +1,13 @@
+#ifndef HeterogeneousCore_AlpakaInterface_interface_Backend_h
+#define HeterogeneousCore_AlpakaInterface_interface_Backend_h
+
+#include <string_view>
+
+// Enumeration whose value EDModules can put in the event
+namespace cms::alpakatools {
+  enum class Backend : short { SerialSync = 0, CudaAsync = 1, ROCmAsync = 2, TbbAsync = 3, size = 4 };
```

Comment:
minor suggestion: would it be better to leave `size` without any assignment, so that it automatically gets the last value plus 1 ?

Reply:
I'd be fine with that